dev.yml

```diff
@@ -9,8 +9,10 @@ up:
       version: 1.14.1
   - custom:
       name: Docker for Mac
-      met?: test -e /Applications/Docker.app
-      meet: echo "Docker not found. Install from https://docs.docker.com/docker-for-mac/"; false
+      met?: test -e /Applications/Docker.app && which docker-compose
+      meet: |
+        echo "Docker not found. Download and Install from https://docs.docker.com/docker-for-mac/install" &&
```

Comment:
should probably add some instructions about installing docker-compose as well now that we're checking for it? or is it included in docker-for-mac now?

Reply:
Upon a successful installation of Docker, `docker-compose` should be installed as well. 

As per the documentation, 
> The Docker Desktop installation includes Docker Engine, Docker CLI client, **Docker Compose**, Notary, Kubernetes, and Credential Helper.

Really, the docker-compose check is a proxy for ensuring the user has _actually installed_ Docker.

Reply:
it may be confusing if the user has an older version of docker that didn't come with docker-compose. Is there a way for us to check the version and assert it is a version with docker-compose included?

Reply:
Ah, this is a good point. 

I've confirmed the following:

We've been enforcing the presence of the Docker Desktop app (transitively `docker-compose`), since this commit https://github.com/Shopify/ghostferry/commit/77e217e63c4534a2956ae50aa1b637876870f8fa. 

If someone has the docker binary installed currently, but not the Docker Desktop app (and possibly not docker-compose), our dev up pipeline in its current state, will fail. This change doesn't particularly influence that behaviour.

---

I'll make sure to edit the copy to provide a hint that `docker-compose` is being checked for.